src/commands/mirror-node.ts

```diff
@@ -349,47 +430,319 @@ export class MirrorNodeCommand extends BaseCommand {
     }
   }
 
+  private enableMirrorNodeTask(): SoloListrTask<AnyListrContext> {
+    return {
+      title: 'Enable mirror-node',
+      task: (_, parentTask): SoloListr<AnyListrContext> =>
+        parentTask.newListr<MirrorNodeDeployContext | MirrorNodeUpgradeContext>(
+          [
+            {
+              title: 'Prepare address book',
```

Comment:
Shouldn't this be conditional based on whether an existing/previous mirror node install is present?